.github/pull_request_template.md

```diff
@@ -32,12 +32,12 @@ What kind of change does this PR introduce?
 
 Please check if your PR fulfills the following requirements:
 
-- [ ] Tested code with current [supported SDKs](../README.md#supported)
-- [ ] Docs have been added/updated which fit [documentation template](https://github.com/nventive/Uno/blob/master/doc/.feature-template.md). (for bug fixes / features)
-- [ ] [Unit Tests and/or UI Tests](doc/articles/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
-- [ ] [Wasm UI Tests](doc/articles/working-with-the-samples-apps.md#running-the-webassembly-ui-tests-snapshots) are not showing unexpected any differences. Validate PR `Screenshots Compare Test Run` results.
+- [ ] Tested code with current [supported SDKs](../README.md#uno-features)
+- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
+- [ ] [Unit Tests and/or UI Tests](../doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
+- [ ] [Wasm UI Tests](../doc/articles/uno-development/working-with-the-samples-apps.md#running-the-webassembly-ui-tests-snapshots) are not showing unexpected any differences. Validate PR `Screenshots Compare Test Run` results.
 - [ ] Contains **NO** breaking changes
-- [ ] Updated the [Release Notes](https://github.com/nventive/Uno/tree/master/doc/ReleaseNotes)
+- [ ] Updated the [Release Notes](../doc/ReleaseNotes)
```

Comment:
Not sure relative links will work there.

Reply:
Relative works in preview mode but I think the PR should be re-worked to use absolute as the links are able to be copied and pasted into a new window during authoring - which is not possible with relative.

Reply:
Replaced